Makefile

```diff
@@ -217,12 +217,22 @@ import-fixtures:  ## import fixtures
 	@$(MANAGE) import_fixtures -v3
 .PHONY: import-fixtures
 
-i18n: ## create/update .po files and compile .mo files used for i18n
+i18n: ## create/update translation files then compile them for both frontend and backend
+i18n: \
+	i18n-compile-back \
+	i18n-compile-front
+.PHONY: i18n
+
+i18n-compile-back: ## create/update .po files and compile .mo files used for i18n
 	@$(MANAGE) makemessages --keep-pot
 	@echo 'Reactivating obsolete strings (allow overriding strings defined in dependencies)'
 	@$(COMPOSE_RUN_APP) find ./ -type f -name django.po -exec sed -i 's/#~ //g' {} \;
 	@$(MANAGE) compilemessages
-.PHONY: i18n
+.PHONY: i18n-compile-back
```

Comment:
should be just "i18n-back" as we do extraction and compilation in the same command (I chose to group everything in one command to simplify things in rsf).

Maybe do the same for frontend translations?